### PR TITLE
Add evm depositor

### PR DESCRIPTION
**Pull request by lmoe**
# Description of change

Introduces a small frontend PoC that allows users to deposit funds to an EVM address. 
Funds will be requested from a faucet.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Enhancement (a non-breaking change which adds functionality)

### CODE REVIEW

File: tools/evm_deposit_frontend/.gitignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+lerna-debug.log*
+
+node_modules
+dist
+dist-ssr
+*.local
+
+# Editor directories and files
+.vscode/*
+!.vscode/extensions.json
```
Comment (jorgemmsilva) [inline tools/evm_deposit_frontend/.gitignore]: not needed?

**Review by jorgemmsilva — APPROVED** [review]

File: tools/evm_deposit_frontend/.gitignore (hunk 1 above)
Comment (lmoe) [reply tools/evm_deposit_frontend/.gitignore]: Default .gitignore from vite, haven't questioned it. 
Usually you never commit the .vscode folder.  Not sure why the extension.json is outruled there though.

Comment (jorgemmsilva) [reply tools/evm_deposit_frontend/.gitignore]: yea, I found it very odd :shrug:

Comment (lmoe) [reply tools/evm_deposit_frontend/.gitignore]: I have removed it. :)

**Review by dessaya — APPROVED** [review]

**Pull request merged**